### ISSUE TITLE
cook_hopsitals gives an error when parsing start date and time

### ISSUE THREAD

**Issue by r-wei** [City-Bureau/city-scrapers#414]
Running `scrapy crawl cook_hospitals` gives this error:

```
Traceback (most recent call last):
  File "~/.virtualenvs/documenters-aggregator/lib/python3.6/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "~/.virtualenvs/documenters-aggregator/lib/python3.6/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "~/.virtualenvs/documenters-aggregator/lib/python3.6/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "~/.virtualenvs/documenters-aggregator/lib/python3.6/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "~/.virtualenvs/documenters-aggregator/lib/python3.6/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "~/projects/city-scrapers/city_scrapers/spiders/cook_hospitals.py", line 39, in parse
    'start': self._parse_start(subitem),
  File "~/projects/city-scrapers/city_scrapers/spiders/cook_hospitals.py", line 99, in _parse_start
    dt = datetime.strptime(start_time, '%B %d, %Y - %H:%M %p')
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/_strptime.py", line 565, in _strptime_datetime
    tt, fraction = _strptime(data_string, format)
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/_strptime.py", line 365, in _strptime
    data_string[found.end():])
ValueError: unconverted data remains:  - 2:00 PM
```

I think this extra " - 2:00 PM" is the end time. If it is, we should capture it in the `end` field.

**Comment by noahkconley** [comment]
@r-wei I can take a look at this

**Comment by r-wei** [comment]
resolved by #448
